.circleci/config.yml

```diff
@@ -48,13 +48,14 @@ jobs:
       - *attach_workspace
       - run:
           name: Run linter
-          command: yarn lint
-      - run:
-          name: Run typecheck
-          command: yarn type-check
-      - run:
-          name: Run tests
-          command: yarn test -w 2
+          command: yarn lint-staged
```

Comment:
staged will be empty, we will want to add a new committed files check or something along those lines

Reply:
Replaced with commands for `eslint` and `prettier` directly